# HG changeset patch
# User L. David Baron <dbaron@dbaron.org>
# Date 1334611049 25200
# Node ID 7c60a6f6a365959a187d25be05c9ec27a85d521a
# Parent  296b90f063d5d9b01be74939869303ba6f6b0527
Fix mistake in manifest file from bug 708175 (causing some tests to be double-run and some not at all).

diff --git a/layout/reftests/font-inflation/reftest.list b/layout/reftests/font-inflation/reftest.list
--- a/layout/reftests/font-inflation/reftest.list
+++ b/layout/reftests/font-inflation/reftest.list
@@ -3,18 +3,18 @@ test-pref(font.size.inflation.emPerLine,
 test-pref(font.size.inflation.emPerLine,15) == text-3.html text-3-ref.html
 test-pref(font.size.inflation.emPerLine,15) == text-4.html text-4-ref.html
 test-pref(font.size.inflation.emPerLine,15) == decoration-1.html decoration-1-ref.html
 test-pref(font.size.inflation.emPerLine,15) == bullet-1.html bullet-1-ref.html
 test-pref(font.size.inflation.emPerLine,15) == bullet-2.html bullet-2-ref.html
 test-pref(font.size.inflation.emPerLine,15) == input-text-1-height.html input-text-1-height-ref.html
 test-pref(font.size.inflation.emPerLine,15) == input-text-1-noheight.html input-text-1-noheight-ref.html
 test-pref(font.size.inflation.emPerLine,15) == input-text-2-height.html input-text-2-height-ref.html
-test-pref(font.size.inflation.emPerLine,15) == input-text-2-height.html input-text-2-height-ref.html
-test-pref(font.size.inflation.emPerLine,15) == input-text-3-noheight.html input-text-3-noheight-ref.html
+test-pref(font.size.inflation.emPerLine,15) == input-text-2-noheight.html input-text-2-noheight-ref.html
+test-pref(font.size.inflation.emPerLine,15) == input-text-3-height.html input-text-3-height-ref.html
 test-pref(font.size.inflation.emPerLine,15) == input-text-3-noheight.html input-text-3-noheight-ref.html
 test-pref(font.size.inflation.emPerLine,15) == textarea-1.html textarea-1-ref.html
 test-pref(font.size.inflation.emPerLine,15) == textarea-2.html textarea-2-ref.html
 test-pref(font.size.inflation.emPerLine,15) == textarea-3.html textarea-3-ref.html
 test-pref(font.size.inflation.emPerLine,15) == css-transform-1.html css-transform-1-ref.html
 test-pref(font.size.inflation.emPerLine,15) == css-transform-2.html css-transform-2-ref.html
 test-pref(font.size.inflation.emPerLine,15) == container-with-clamping.html container-with-clamping-ref.html
 test-pref(font.size.inflation.emPerLine,15) load video-1.html
